test-cypress/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM cypress/base:12
+
+COPY . /workbench/tests-cypress/
+
+WORKDIR /workbench/tests-cypress/
+
+RUN npm install && npm install --save-dev cypress
```

Comment:
`&& npm install --save-dev cypress`

Cypress should already be included in the dependencies.

Reply:
Yeah, but it does not work without this for some reason, I'll check why

Reply:
Yep, doesn't work, gives me errors
Cypress executable not found at: /root/.cache/Cypress/3.8.3/Cypress/Cypress